src/util.h

```diff
@@ -434,23 +434,23 @@ void RenameThreadPool(ctpl::thread_pool& tp, const char* baseName);
 /**
  * .. and a wrapper that just calls func once
  */
-template <typename Callable> void TraceThread(const char* name,  Callable func)
+template <typename Callable> void TraceThread(std::string name,  Callable func)
```

Comment:
Let's keep `name` immutable, just in case
```suggestion
template <typename Callable> void TraceThread(const std::string name,  Callable func)
```